### PR TITLE
Use FQCN for services and documents

### LINKED ISSUE #537: Use FQCNs as service identifiers
Leftover from #535

### PR DESCRIPTION
**Pull request by malarzm**
Fixes #537

### CODE REVIEW

File: Resources/doc/cookbook/registration_form.rst [hunk 1]
```diff
@@ -254,7 +254,7 @@ the validation and saves the data into MongoDB:
 
     public function createAction(Request $request)
     {
-        $dm = $this->get('doctrine_mongodb')->getManager();
+        $dm = $this->get(DocumentManager::class);
```
Comment (malarzm) [inline Resources/doc/cookbook/registration_form.rst]: I wasn't fixing `$this->get` calls just yet, but I can do it now, just need to figure out how to show that we're injecting DocumentManager. Or should we do method injection?

Comment (alcaeus) [reply Resources/doc/cookbook/registration_form.rst]: I'm all for using method injection: it's enabled by default and it avoids having to add a constructor for DI purposes to the code samples.

Comment (malarzm) [reply Resources/doc/cookbook/registration_form.rst]: Done 👍

**Comment by alcaeus** [comment]
Thanks @malarzm!

**Pull request merged**